components/modals/documentation-modal.tsx

```
@@ -48,10 +46,10 @@ export function DocumentationModalInner({
                 <div className="flex gap-1">
                     <a
                         className="hover:underline underline-offset-4"
-                        href="https://w3id.org/ro/crate/1.1"
+                        href="https://w3id.org/ro/crate/1.2"
                         target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)